raven/processes/wps_nalcms_zonal_stats_raster.py

```diff
@@ -0,0 +1,239 @@
+import json
+import logging
+import tempfile
+from collections import defaultdict
+from pathlib import Path
+
+from pywps import ComplexOutput
+from pywps import Process, FORMATS
+from pywps.inout.outputs import MetaLink4, MetaFile
+from rasterstats import zonal_stats
+
+from raven.utilities import gis
+from raven.utils import (
+    archive_sniffer,
+    crs_sniffer,
+    generic_vector_reproject,
+    raster_datatype_sniffer,
+    single_file_check,
+    zonalstats_raster_file,
+)
+from . import wpsio as wio
+
+LOGGER = logging.getLogger("PYWPS")
+
+TRUE_CATEGORIES = {
```

Comment:
That's already defined elsewhere, no ? Why not import it ?

Reply:
Good point. Will do!